addons/web/static/src/core/notifications/notification.scss

```diff
@@ -1,47 +1,13 @@
-$notification-width: 320px;
-$notification-text-color: #212529;
-$notification-title-text-color: #6c757d;
-
 // Colors of the notification are not managed by this file.
 // It is handled by the other classes defining a background. (like danger, warning, etc.)
 // The notification just add overlays.
 
 .o_notification_manager {
-  @include o-position-absolute(3.5rem, 0);
-  position: fixed;
+  @include o-position-absolute($o-navbar-height * 1.15, map-get($spacers, 2), $left: calc(100vw - #{$toast-max-width}));
   z-index: ($zindex-modal + $zindex-popover) / 2;
```

Comment:
Can I ask you to fix this too? Not sure about this rule. Notification manager is on top of modal but below of popover. This calculation doesn't make sense to me...

Reply:
Not sure either, haven't touched that.
Should the notification go below of modal and above popover?

Reply:
@xlu-odoo after discussion with dbo, we should remove this calculation and replace by a variable. Lets keep the actuel behavior except for the burger menu: 
`$zindex-dropdown:                   1000 !default;
$zindex-sticky:                     1020 !default;
$zindex-fixed:                      1030 !default;
$zindex-modal-backdrop:             1040 !default;
$zindex-modal:                      1050 !default;
$zindex-burger-menu:                1052 !default;
$zindex-notifcation:                1055 !default;
$zindex-popover:                    1060 !default;
$zindex-tooltip:                    1070 !default;`